### PR TITLE
Fix wrong iam_addictive variable input

### PR DESCRIPTION
**Pull request by eturan**
iam_additive variable from Project module expects { "roles" = list(string) } input.
Therefore, I believe the issue #174 is till active

### CODE REVIEW

**Comment by ludoo** [comment]
failing test in environments is unrelated BTW

File: modules/project/README.md [hunk 1]
```diff
@@ -36,20 +36,12 @@ module "project" {
...
-	    "roles/owner",
-    ],
-    "group:usergroup_gcp_privilege_access@lemonadeinc.io" = [
-	    "roles/editor"
-    ],
-    "group:engineering@lemonadeinc.io" = [
-	    "roles/pubsub.subscriber",
-	    "roles/storage.objectViewer"
-    ],
+    "roles/viewer"               = ["group:usergroup_watermlon_experimentation@lemonadeinc.io],
```
Comment (ludoo) [inline modules/project/README.md:39]: You're missing closing quotes and the example is failing tests :)

**Comment by ludoo** [comment]
Emre, thanks for this! Can you rebase to master so the environments test is fixed, and add closing quotes in the README example, so the examples tests are happy?

**Comment by ludoo** [comment]
moving to a separate branch to apply fixes and merge to master, thanks for this!

**Comment by eturan** [comment]
sorry for the late response @ludoo it was my birthday and I was not allowed to 'code' :). Thanks for the approving the changes. Have a great one!

**Comment by ludoo** [comment]
No worries, and late happy birthday! :+1: on the no-code policy of course!

**Pull request merged**